io.github.TTimo.GtkRadiant.json

```diff
@@ -0,0 +1,220 @@
+{
+  "app-id": "io.github.TTimo.GtkRadiant",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
```

Comment:
this version of the runtime as been **EOL for a few years**.

Reply:
The current source doesn't build out of the box against newer SDKs. A bit of investigation suggests it would take some non trivial refactoring and developer time to modernize.

GtkRadiant was written 20+ years ago against the Gtk 2 libraries. It uses an obsolete module (gtkglext) that depends on more obsolete pieces (pangox).

It has been increasingly difficult to build GtkR on modern distributions; we are looking to leverage flatpak's older SDKs and Platforms to help with this problem.

Reply:
OK, I'll rephrase it. Use a supported runtime if you want this to be accepted. You probably will need to bring some dependencies like Python2 (it's a one line addition to the manifest with shared-modules).

Reply:
Interesting. Thanks for the feedback. We'll have to see internally if it's even worth it for us being on flathub then.

Reply:
Interesting. Thanks for the feedback. We'll have to see internally if it's even worth it for us being on flathub then.